docs/classes/server.md

```diff
@@ -3,71 +3,99 @@ title: Server
 permalink: /classes/server
 ---
 
-{:.warning}
-> This page needs an update.
-
 * Table of Contents
 {:toc}
 
 The `Server` instance controls the whole listening and dispatches the parsed requests.
 
-## `attach(ServerObserver)`
+The `Server` class has [custom debug output](http://php.net/manual/en/language.oop5.magic.php#object.debuginfo): 
```

Comment:
I wouldn't remove it completely, but remove the output and add: custom debug output, which isn't part of the public API and which shouldn't be relied upon.

Reply:
Fixed.